src/api/reservations.data.ts

```diff
@@ -1,6 +1,6 @@
 import dayjs, { Dayjs } from 'dayjs';
-
-interface Reservation {
+//todo: split model and date
```

Comment:
model and data :)

The reason is this `reservations.data.ts` file should eventually be removed and replaced with an API while the model will stay.